minvws/nl-kat-coordination
# Feature: Render human readable ooi names in onboarding flow

### Changes

In the steps where the user accepts their clearance level and sets the OOI clearance level the OOI ID was rendered, instead of the human readable name, which was prefered. This PR solves this and renders the name as requested.

### Issue link

Closes #3441 

### Demo

![image](https://github.com/user-attachments/assets/d2c98fcb-1bf9-4ba4-85b0-2354486be348)

### QA notes

Add an OOI in the onboarding flow and verify that the rendered name is no longer <ooi_name>|<network>|<ooi_type> but <ooi_name> @ <network>
---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
